packages/default-theme/components/SwMegaMenu.vue

```diff
@@ -8,7 +8,7 @@
       >
         <nuxt-link
           class="sf-header__link"
-          :to="getCategoryUrl(subcategory)"
+          :to="$i18n.path(getCategoryUrl(subcategory))"
```

Comment:
should we translate URLs as well? shouldn't it come from API after the context changes? or I don't get it :smile:

Reply:
URLs do not change, this method changes links to current locale link :)

Reply:
aa I see :) great!